concordium-std-derive/CHANGELOG.md

```diff
@@ -8,3 +8,4 @@
 
 - Add macros for deriving error implementations.
 - Make derive macros slightly more hygienic.
+- Enable renaming of enum variants and field names for the schema.
```

Comment:
It would be useful to add a bit more detail here, specifically what the new features are (rename attribute, ...)